sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -601,9 +616,9 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
     checkEvaluation(Slice(a0, Literal(4), Literal(10)), Seq(4, 5, 6))
     checkEvaluation(Slice(a0, Literal(-1), Literal(2)), Seq(6))
     checkExceptionInExpression[RuntimeException](Slice(a0, Literal(1), Literal(-1)),
```

Comment:
Could you use `checkErrorInExpression` to check the error class and its parameterers.